Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -1725,12 +1725,18 @@ extension SourceKitServer {
     }
 
     let resolved = self.extractIndexedOccurrences(symbols: symbols, index: index, useLocalFallback: true) {
```

Comment:
We could just skip this if the symbol isn't dynamic - the index will be at most up to date as the location in the module, or worse. Though that would change if background indexing was added (or preparation, if we go that route instead).

Reply:
Ah, I just noticed this is on SourceKitServer and not the language service. Would still be nice to skip for Swift IMO. As for clangd not returning dynamic, that's something we could change (the request we're using is one we specifically added for this use case - https://reviews.llvm.org/D54799). But in the meantime, I'd default to returning overrides if `isDynamic` isn't set.

Reply:
Changed it so that we don’t do an index lookup if `SymbolRequest` returns a location.